bakery/src/cli/execute.js

```diff
@@ -125,13 +126,36 @@ const flyExecute = async (cmdArgs, { image, persist }) => {
       output: 'silent'
     })
 
-    console.log('syncing')
-    const sync = spawn('fly', [
-      'sync',
-      '-c', 'http://localhost:8080'
-    ], { stdio: 'inherit' })
-    children.push(sync)
-    await completion(sync)
+    try {
+      console.log('syncing')
+      const sync = spawn('fly', [
+        'sync',
+        '-c', 'http://localhost:8080'
+      ], { stdio: 'inherit' })
+      children.push(sync)
+      await completion(sync)
+    } catch (err) {
```

Comment:
What if user does NOT have fly installed despite the installation instructions to do so. Should we just download the correct fly version from the concourse container and use it anyways?

Reply:
I think this would be a nice user experience. Without `which` resolving would you just drop it in the current directly and add that to the path for the CLI?

Reply:
I like the idea of using `.local/bin` in `$HOME`

Reply:
Now uses `$HOME/.local/bin/bakery-cli-fly` as a last resort, but will prefer `fly` on the path and sync it if needed.